yt/utilities/flagging_methods.py

```diff
@@ -8,13 +8,16 @@
 class RegisteredFlaggingMethod(type):
```

Comment:
Is there a reason this class isn't removed in https://github.com/yt-project/yt/pull/2823 ?

Reply:
Yes -- it's super duper localized, and a candidate for removal at a later time.